### PR TITLE
Add EC2_TAGS option (collect_ec2_tags)

### PR DESCRIPTION
**Pull request by homerovalle**
This PR adds support for enabling "collect_ec2_tags" on the agent using an environment variable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ e.g. the first version of the Docker image that will bundle the Datadog Agent 5.
 A few parameters can be changed with environment variables.
 
 * `TAGS` set host tags. Add `-e TAGS="simple-tag-0,tag-key-1:tag-value-1"` to use [simple-tag-0, tag-key-1:tag-value-1] as host tags.
+* `EC2_TAGS` set EC2 host tags. Add `-e EC2_TAGS=yes` to use EC2 custom host tags.
```
Comment (hkaj) [inline README.md]: Could you add a note stating that this requires the instance to have an IAM role to work? Something similar to https://github.com/DataDog/dd-agent/blob/5.7.4/datadog.conf.example#L33 would be perfect

**Comment by hkaj** [comment]
Hi @homerovalle 
Thanks for the contribution! I added a minor comment, once it's addressed this is good to go.

Thanks again

**Comment by homerovalle** [comment]
Fixed merge conflict, should be ready to go! Thanks!

**Comment by hkaj** [comment]
thank you @homerovalle 🎉

**Pull request merged**